<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClickGameWindow</class>
 <widget class="QMainWindow" name="ClickGameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Click me</string>
    </property>
   </widget>
   <widget class="QLabel" name="counterLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Clicks: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="winLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>You win</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
